components/atoms/ToggleOption/toggle-option.tsx

```diff
@@ -10,6 +10,15 @@ const ToggleOption = ({ optionText, withIcon, checked, handleToggle }: ToogleOpt
   return (
     <div
```

Comment:
The same comment as the other ones abou div to buttons and similar situations in the rest of the files in the PR.